modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/internal/content/processor/test/DefaultExportImportContentProcessorTest.java

```diff
@@ -581,6 +581,40 @@ public void testExportLinksToUserLayouts() throws Exception {
 		_assertLinksToLayouts(content, publicLayout, publicLayout.getGroupId());
 	}
 
+	@Test
+	public void testExportUUIDDLReference() throws Exception {
+		FileEntry fileEntry = DLAppLocalServiceUtil.addFileEntry(
+			null, TestPropsValues.getUserId(), _stagingGroup.getGroupId(),
+			DLFolderConstants.DEFAULT_PARENT_FOLDER_ID,
+			"00000000-0000-0000-0000-000000000000.txt", ContentTypes.TEXT_PLAIN,
+			TestDataConstants.TEST_BYTE_ARRAY, null, null,
+			ServiceContextTestUtil.getServiceContext(
+				_stagingGroup.getGroupId(), TestPropsValues.getUserId()));
+
+		ThumbnailCapability thumbnailCapability =
+			fileEntry.getRepositoryCapability(ThumbnailCapability.class);
+
+		fileEntry = thumbnailCapability.setLargeImageId(
+			fileEntry, fileEntry.getFileEntryId());
+
+		_portletDataContextExport.setZipWriter(new TestReaderWriter());
+
+		String content = _replaceParameters(
+			_getContent("dl_references.txt"), fileEntry);
+
+		boolean valid = true;
+
+		try {
+			_exportImportContentProcessor.validateContentReferences(
+				_stagingGroup.getGroupId(), content);
+		}
+		catch (Exception exception) {
+			valid = false;
+		}
```

Comment:
Why not let the exception be thrown?  You can remove the catch block and the assertion, and the test will automatically fail if an exn is thrown.

Reply:
@NemethNorbert I understand. The problem I see with this is that this doesn't follow our common practice. I think this will have more chances of being  merged if you simply let the exception be thrown and the test to fail directly. The test is small enough so that the failures is not too cryptic.

Reply:
Hey @adolfopa ,

I removed the try catch block and the assertion and made a force push.

Thanks